CMakeLists.txt

```diff
@@ -556,7 +560,7 @@ ADD_CUSTOM_TARGET(mops2 DEPENDS ${PCH})
 subdirs(
       mscore  awl bww2mxml share midi audiofile fluid mstyle libmscore synthesizer
       effects thirdparty/rtf2html thirdparty/diff thirdparty/beatroot
-      thirdparty/qzip thirdparty/kQOAuth
+      thirdparty/qzip thirdparty/kQOAuth thirdparty/breakpad
```

Comment:
it would probably be good to add an option for breakpad, just like OMR, ZERBERUS etc... below.